app/lib/three_scale/semantic_form_builder.rb

```diff
@@ -2,7 +2,7 @@ module ThreeScale
   class SemanticFormBuilder < ::Formtastic::FormBuilder
     include ThreeScale::BotProtection::Form
 
-    delegate :tag, to: :template
+    delegate :tag, :site_account, :controller, to: :template
```

Comment:
This comment is not about this line, but about this file :upside_down_face: 

So, at the bottom of this we have a method:
```
    # Just adds fields from bot protection module
    def bot_protection
      bot_protection_inputs
    end
```

I am thinking maybe we could remove that, and make `bot_protection_inputs` public? :thinking: It's currently used just in one place, it seems, and we can just use `.bot_protection_inputs` there, WDYT?
This is just to reduce complexity, but I don't know, maybe it's not worth it.

Reply:
I replaced this by an alias. I don'¡t think it's worth renaming all references in tests and UI for this.

Reply:
Yeah, but from I can tell it's just a couple of uses (at least I couldn't find any more)